### ISSUE TITLE
Admin Page: conflict with WP skip links results in blank pages

### ISSUE THREAD

**Issue by afercia** [Automattic/jetpack#6634]
#### Steps to reproduce the issue

Using the keyboard to navigate, just use the WordPress "Skip to main content" or "Skip to toolbar" links.

#### What I expected

To see the web page content.

#### What happened instead
Page is blank.

Since Jetpack uses its own URL fragment identifier format for navigation and browser's history, activating the WordPress skip links makes the resulting URL something like:
```
?page=jetpack#/wpbody-content
?page=jetpack#/wp-toolbar
```

of course, this fails since there are no corresponding routes.

**Comment by eliorivero** [comment]
Thanks for the report! I've addressed this in #6642.

**Comment by eliorivero** [comment]
Closed in #6642
